GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -166,6 +166,9 @@ void Mesh::generateVAO() {
         return;
     }
 
+    GLuint vaoID_ = 0;
+	GLuint triangle_vboID_, vert_vboID_, norm_vboID_, tex_vboID_;
```

Comment:
Maybe. I’ve had success formatting .h and .cpp files – it’s the *_jni.cpp files that don’t work well.

From: Guodong Rong [mailto:notifications@github.com]
Sent: Thursday, May 28, 2015 11:29 AM
To: Samsung/GearVRf
Cc: Jon Shemitz
Subject: Re: [GearVRf] Fixed the bug of resuing mesh with different shaders. (#73)


In GVRf/Framework/jni/objects/mesh.cpp<https://github.com/Samsung/GearVRf/pull/73#discussion_r31263207>:

> @@ -166,6 +166,9 @@ void Mesh::generateVAO() {

>          return;

>      }

>

> +    GLuint vaoID_ = 0;

> +     GLuint triangle_vboID_, vert_vboID_, norm_vboID_, tex_vboID_;

I do not see any indenting on my PC. And it seems Shift+Ctrl+F does not work correctly for CPP files. Maybe my formatter setting has any problems?

—
Reply to this email directly or view it on GitHub<https://github.com/Samsung/GearVRf/pull/73/files#r31263207>.